lib/Dialects/VectorExt/Transform/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
+set(LLVM_OPTIONAL_SOURCES
```

Comment:
You can add `PARTIAL_SOURCES_INTENDED` to `add_mlir_library` and avoid listing all files here (or, better, have one library per directory).

Reply:
I'll leave this as is because the file will be removed entirely very soon. But this is very useful for some of my other changes...